.github/workflows/runtime-compatibility.yml

```diff
@@ -0,0 +1,114 @@
+name: Runtime Compatibility Test
+on:
+  workflow_run:
+    workflows: ["Release Runtime Build"]
+    types:
+      - completed
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: 'Workflow run ID to download artifacts from'
+        required: true
+        type: string
+
+jobs:
+  fetch-client-info:
+    # Only run if workflow_run was successful OR if manually triggered
+    if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-latest
+    outputs:
+      client_tag: ${{ steps.parse-releases.outputs.client_tag }}
+      client_asset_url: ${{ steps.parse-releases.outputs.client_asset_url }}
+    steps:
+      - name: Fetch and parse client releases
+        id: parse-releases
+        run: |
+          # Retry API calls to handle rate limiting
+          for i in {1..3}; do
+            RELEASES=$(curl -s https://api.github.com/repos/AstarNetwork/Astar/releases)
+            if echo "$RELEASES" | jq empty 2>/dev/null; then
+              break
+            fi
+            sleep $((i * 5))
+          done
+          
+          # Get latest client tag (v*.*.* format)
```

Comment:
not sure about this, the idea is to test new runtime with old clients, if we release new client then the test will always pass

Reply:
Is there something better we can do though?
Test last two releases? Seems like an overkill.

Reply:
Good point @ermalkaleci, the workflow checks compatibility for the 2 latest clients now. Since they're non-gating checks, this only provides extra infos.